index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
Instead of `word-break: break-all;` you could try:

```css
h4 a {
    -webkit-hyphens: auto;
    -ms-hyphens: auto;
    hyphens: auto;
}
```

This might also require `<html lang="en">` on some browsers.

<https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens>